### ISSUE TITLE
Handle "Cannot join channel (+r)" messages

### ISSUE THREAD

**Issue by osa1** [osa1/tiny#32]
Debug output:

```
[Sat, 04 Nov 2017 08:56:37 ] Ignoring numeric reply msg:
Pfx: Server("tolkien.freenode.net"), num: 477, args: ["osa1_", "#tiny", "Cannot join channel (+r) - you need to be identified with services"]
```

**Comment by osa1** [comment]
We should find an easier way of handling these error messages. RFC 2812 lists dozens of errors but we only handle a small subset of them, handling all of them in current style would require hundreds of lines of code.

**Comment by osa1** [comment]
Already fixed with 0beed80.
